tests/test_unit.py

```diff
@@ -68,3 +68,28 @@ def test_plural(self):
         validator = NumericValidator(1)
         with pytest.raises(ValidationError, match="^Pole musí mít právě 1 znak$"):
             validator(form, form.field)
+
+
+class TestCapitalizer:
+    @pytest.mark.parametrize(
+        ("input", "expected"),
+        (
+            ("karlov", "karlov"),
+            ("Karlov", "Karlov"),
+            ("KARLOV", "Karlov"),
+            ("Velké KARLOVICE", "Velké Karlovice"),
+            ("velké karlovice", "velké karlovice"),
+            ("VELKÉ karlovice", "Velké karlovice"),
+            ("VELKÉ KARLOVICE", "Velké Karlovice"),
+            ("a b c d", "a b c d"),
+            ("A B C D", "A B C D"),
+            ("a B c D", "a B c D"),
+            ("U LÍPY", "U Lípy"),
+            ("u lípy", "u lípy"),
+            ("U Lípy", "U Lípy"),
+        ),
+    )
+    def test_capitalize(self, testapp, input, expected):
```

Comment:
Přemýšlím, jestli nechceme i test těch, co se převedou „špatně“ s nějakou peprnou poznámkou. Případně jako _xfail_.

Reply:
Přidám `("FRÝDLANT NAD OSTRAVICÍ", "Frýdlant Nad Ostravicí"),` což je lepší než originál, ale není to správně.